RELEASE.md

```diff
@@ -14,6 +14,7 @@
 ## Breaking changes to the API
 * Simplify the Kedro template in `run.py` with the introduction of `KedroContext` class.
 * Merged `FilepathVersionMixIn` and `S3VersionMixIn` under one abstract class `AbstractVersionedDataSet` which extends`AbstractDataSet`.
+* `src/environment.yml` will be required to run `kedro install --conda`
```

Comment:
Could we make this an opt-in feature and not have this as a breaking change?

Reply:
@idanov - I'm not really sure by what you mean by an opt-in-feature. Are there any examples you could show me?

To me the difference between running ```kedro install``` and ```kedro install --conda``` requires opting in as you've chosen to add the flag.

Reply:
@idanov can keep me honest here, but I presume he means checking the file exists, instead of checking a flag is provided.
sth like 
```
if (Path.cwd() / "src" / "environment.yml").is_file()
```
and then the `--conda` flag can be removed

Reply:
Ok cool. I'll go under the assumption that's the solution. I'll get it wrapped up on Monday (the code change and update the docs)